python/lsst/meas/extensions/shapeHSM/_hsm_shape.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, config, name, schema, metadata, logName=None):
         self.hasDeblendKey = len(config.deblendNChild) > 0
 
         if self.hasDeblendKey:
-            self.deblendKey = schema[config.deblendNChild]
+            self.deblendKey = schema.find(config.deblendNChild).key
```

Comment:
Is this `schema.find` approach considered standard?

Reply:
Was there a problem with how it was prior to the change?

Reply:
Yes, it was causing `'lsst.afw.table._table.SubSchema' object has no attribute 'get'` with `self.hasDeblendKey==True`.

Reply:
Yes, this was incorrect before. Either what @enourbakhsh did here, or `schema[name].asKey()` is also equivalent.

Reply:
The `schema.find(name).key` method is thanks to Dan's input. If they're both equivalent, I personally find the `schema[name].asKey()` method easier to read/digest.